instances/widgets.treasury-factory.near/widget/components/HistoryStatus.jsx

```diff
@@ -43,6 +48,8 @@ return (
       <div className="reject bold">Rejected</div>
     ) : proposalStatus === "Failed" ? (
       <div className="failed bold">Failed</div>
+    ) : proposalStatus === "Active" ? (
+      <div className="active bold">Active</div>
```

Comment:
why is there an active proposal in `HistoryStatus`? And I don't see any active status in [contract](https://github.com/near-daos/sputnik-dao-contract/blob/main/sputnikdao2/src/proposals.rs#L19) 
Please add a comment about what this value is?

Reply:
it's just for iindicating that proposal returns to iinitial state. @FREZZZBE is it correct?
<img width="814" alt="Screenshot 2025-04-08 at 20 28 45" src="https://github.com/user-attachments/assets/4395d207-b6ff-425c-bf1a-3712dc995ea7" />